tests/requirements-libraries.txt

```diff
@@ -96,3 +96,7 @@ uvloop==0.11.3; python_version >= '3.5' and sys_platform != 'win32'
 # Python 3 only packages
 # ----------------------
 pylint==2.2.2; python_version != '2.7'
+
+# Windows 64 bits only
+# --------------------
+itk==4.13.1.post1; sys_platform != 'win32'
```

Comment:
I'm not sure whether we shall add this *huge* package to the testsuite. This will add about 114MB of download files. This is nine times the size of numpy.

Reply:
I removed this requirement. I left the test for users to be able to run it manually if they want to test ITK with PyInstaller. Since it is decorated with the standard PyInstaller test decorator, it will be skipped by default (since ITK will not be installed). In order for the test to run, the user will have to manually install this requirement.